kits/sparktrace/prompts/sparktrace-reporter_generate-json_user.md

```
@@ -0,0 +1,4 @@
+Full investigation to synthesize (JSON — Investigation):
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)